src/components/CreateCourse/CreateCourse.tsx

```diff
@@ -45,11 +82,19 @@ export const CreateCourse: React.FC = () => {
 	const [authorName, setAuthorName] = useState('');
 	const [authorNameErrorLabelValue, setAuthorNameErrorLabelValue] =
 		useState('');
-	const [authorList, setAuthorList] = useState(mockedAuthorsList);
+	const [authorList, setAuthorList] = useState([]);
 	const [courseAuthorList, setCourseAuthorList] = useState([]);
 	const [courseAuthorListErrorLabelValue, setCourseAuthorListErrorLabelValue] =
 		useState('');
 
+	const authors = useSelector(
+		(state: RootState) => state.authorReducer.authors
```

Comment:
create a selectors file in store for authors using createSelector (see below the link)